src/app.js

```diff
@@ -702,15 +1464,19 @@ gpii.app.menu.generateMenuTemplate = function (/* all the items in the menu */)
     var menuTemplate = [];
     fluid.each(arguments, function (item) {
         if (item) {
-            menuTemplate.push(item);
+            if (Array.isArray(item)) {
```

Comment:
Looks like this could just be a call to fluid.flatten(fluid.makeArray(arguments))